en/docs/learn/consume-api/manage-subscription/subscribe-to-an-api.md

```diff
@@ -72,7 +72,23 @@ If you already have an existing application, follow the instructions below to su
      You can see the subscriptions list in API credentials section.
 
      [![Subscribe to new app]({{base_path}}/assets/img/learn/subscription-list.png)]({{base_path}}/assets/img/learn/subscription-list.png)
+
+## Update the subscription tier
+
+1.  Sign in to the WSO2 API Developer Portal (`https://<hostname>:<port>/devportal`). Click on **Applications** and select the relevant application. From the left navigational menu click on the **Subscriptions** tab.
```

Comment:
"From the left navigational menu click on the **Subscriptions** tab." - Based on the common writing standard that we follow we use images to show the placement of the menu/tab. So this should change to "Click Subscriptions" as the image is there to show the user the placement of the tab.